js/index.js

```diff
@@ -0,0 +1,345 @@
+      if (listItems === null || listItems[0] === '') {
+        listItems = [name];
+      //otherwise
+    } else {
+        //add item
+        listItems.push(name);
+      }
+      //update list items on local storage
+      setListItem($('.listNames ul').find("input:radio:checked").val(), listItems);
+    }
+    //prevent page reload
+    event.preventDefault();
+  });
+  /*
+  * Function that deals with showing list items based on list name
+  */
+  $('.listNames ul').click(function() {
+    //empty list items
+    $('.listItems ul').empty();
+    //get all list items under a line name
+    var listItems = getListsItems($('.listNames ul').find("input:checked").val());
+    //as long as listitems is valid
+    if (listItems !== null && listItems[0] !== '') {
+      //show list
+      parseList(listItems, addItem);
+      var listItems2 = getListsItems($('.listNames ul').find("input:checked").val().concat('.Checked'));
+      //add highlighting to list items
+      parseChecked(listItems2);
+    }
+  });
+  /*
+  * Function that deals with toggling of highlight and checked on click of both list and list items
+  */
+  $('ul').on('click', 'li', function(){
+    //if this is a list
+    if ($(this).find('input').is(':radio')) {
+      //remove highlighted from all lists
+      $('li').each(function() {
+        $(this).toggleClass('highlight', false);
+      });
+    }
+    //if this is a list and not checked, then check
+    if ($(this).find("input:radio").val()){
+      if (!$(this).find('input').prop('checked')){
+        $(this).find('input').prop('checked', true);
+      }
+      //add highlighting to item
+      $(this).toggleClass('highlight', this.checked);
+    //otherwise clicked is list item
+    } else {
+      //get list of checked items
+      var checked = getListsItems($('.listNames ul').find("input:checked").val().concat('.Checked'));
+      //if item is checked
+      if ($(this).find('input').prop('checked')){
+        //remove checked from item and remove from checked list
+        $(this).find('input').prop('checked', false);
+        checked[$(this).index()] = '';
+        //otherwise
+      } else {
+        //add checked and add it to checked list
+        $(this).find('input').prop('checked', true);
+        checked[$(this).index()] = 'checked';
+      }
+      //highlight item and add to list of checked on local storage
+      $(this).toggleClass('highlight', this.checked);
+      setListItem($('.listNames ul').find("input:checked").val().concat('.Checked'), checked);
+    }
+  });
+  /*
+  * Function to delete list
+  */
+  function deleteList() {
+    //if the following is a list or list item
+    if ($(this).find('input').is(':radio')) {
+      //iterate through each
+      $(this).parent().each(function() {
+        //remove highlighting for each
+        $(this).toggleClass('highlight', false);
+      });
+    }
+    //if the following is a list or list item
+    if ($(this).prev().find('input').is(':radio'))
```

Comment:
Be consistent with the use of brackets in your if statements. Not having them is ok, just make sure you always or never, not sometimes